test/battle/weather/hail.c

```diff
@@ -32,18 +32,20 @@ SINGLE_BATTLE_TEST("Hail damage does not affect Ice-type Pokémon")
 
 SINGLE_BATTLE_TEST("Hail fails if Desolate Land is active")
```

Comment:
Please update the name of the test to account for the change.

Reply:
done